src/write_outputs/capacity_reserve_margin/write_capacity_value.jl

```diff
@@ -34,48 +34,90 @@ function write_capacity_value(path::AbstractString, inputs::Dict, setup::Dict, E
 		AC_CHARGE_EX = intersect(inputs["VS_STOR_AC_CHARGE"], VRE_STOR_EX)
 		dfVRE_STOR = inputs["dfVRE_STOR"]
 	end
-	
-	totalcap = repeat((value.(EP[:eTotalCap])), 1, T)
+
+    crm_derate(i, y::Vector{Int}) = dfGen[y, Symbol("CapRes_$i")]'
+    max_power(t::Vector{Int}, y::Vector{Int}) = inputs["pP_Max"][y, t]'
+    total_cap(resources::Vector{Int}) = eTotalCap[resources]'
```

Comment:
You could use `y` instead of `resources` for consistency with the previous definitions
```suggestion
    total_cap(y::Vector{Int}) = eTotalCap[y]'
```